packages/roosterjs-editor-api/lib/cursor/cacheGetCursorEventData.ts

```diff
@@ -4,14 +4,23 @@ import { PluginEvent } from 'roosterjs-editor-types';
 
 const EVENTDATACACHE_CURSORDATA = 'CURSORDATA';
 
-// Read CursorData from plugin event cache. If not, create one
+/**
+ * Read CursorData from plugin event cache. If not, create one
+ * @param event The plugin event
```

Comment:
Explain why we pass in event here, and what will happen if event is null.

Reply:
Added